test.js

```diff
@@ -21,6 +20,38 @@ test('main', async t => {
 	}));
 });
 
+test('limits after pause', async t => {
+	const limit = 10;
+	const interval = 1010;
+	const throttled = pThrottle({limit, interval})(() => Date.now());
+
+	const promises = [];
+
+	await throttled(0);
+
+	await new Promise(resolve => {
+		setTimeout(resolve, interval / 2);
+	});
+
+	const start = Date.now();
+
+	for (let i = 0; i < 11; i++) {
+		promises.push(throttled(i));
+	}
+
+	const results = await Promise.all(promises);
+
+	results.forEach((executed, index) => {
```

Comment:
Use a `for-of` loop (just my preference).

Reply:
Ok